x-pack/plugins/enterprise_search/public/applications/app_search/components/analytics/analytics_logic.test.ts

```diff
@@ -0,0 +1,308 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+import { LogicMounter } from '../../../__mocks__/kea.mock';
+
+jest.mock('../../../shared/kibana', () => ({
+  KibanaLogic: { values: { history: { location: { search: '' } } } },
+}));
+import { KibanaLogic } from '../../../shared/kibana';
+
+jest.mock('../../../shared/http', () => ({
+  HttpLogic: { values: { http: { get: jest.fn() } } },
+}));
+import { HttpLogic } from '../../../shared/http';
+
+jest.mock('../../../shared/flash_messages', () => ({
+  flashAPIErrors: jest.fn(),
+}));
+import { flashAPIErrors } from '../../../shared/flash_messages';
+
+jest.mock('../engine', () => ({
+  EngineLogic: { values: { engineName: 'test-engine' } },
+}));
+
+import { AnalyticsLogic } from './';
+
+describe('AnalyticsLogic', () => {
+  const DEFAULT_VALUES = {
+    dataLoading: true,
+    analyticsUnavailable: false,
+  };
+
+  const MOCK_TOP_QUERIES = [
```

Comment:
I've found value in typing these sorts of default values for tests, it can catch stuff when the properties of these objects change

Reply:
I'm not 100% convinced of this - our Kea logic should be type-checking for us and alerting if we have type issues. Not sure if `ent-search` has been updated to use `makeLogicType`, but in Kibana I definitely know it type checks my test files.